token-lending/program/tests/liquidate_obligation.rs

```diff
@@ -40,7 +40,8 @@ async fn test_success() {
     const USDC_RESERVE_LIQUIDITY_FRACTIONAL: u64 = 2 * USDC_BORROW_AMOUNT_FRACTIONAL;
 
     let user_accounts_owner = Keypair::new();
-    let user_transfer_authority = Keypair::new();
+    let user_transfer_authority =
```

Comment:
have you tested with a non whitelisted liquidator?

Reply:
good point yeh let me add